### ISSUE TITLE
Adjust accessibility statement after fixes

### ISSUE THREAD

**Issue by selfthinker** [alphagov/govuk-brand-guidelines#230]
Change the compliance status on the accessibility statement and remove the 'Non-accessible content' section when the outstanding WCAG fails have been fixed.

This depends on the following issues being fixed:

* #193
* #229
* #218
* #219
* #226

What needs to be done is in TODO comments within accessibility-statement.md:

* [x] Change the 'Compliance status' section to say the website is fully compliant.
* [x] Remove the whole 'Non-accessible content' section.
* [x] Add the sentence with the date when the statement was last reviewed in the 'Preparation of this accessibility statement' section.
